src/main/features/response/form/models/responseType.ts

```diff
@@ -13,9 +13,9 @@ export class ResponseType {
 
   static all (): ResponseType[] {
     return [
-      ResponseType.OWE_ALL_PAID_NONE,
-      ResponseType.OWE_SOME_PAID_NONE,
-      ResponseType.OWE_NONE
+      ResponseType.FULL_ADMISSION,
```

Comment:
The names has been confirmed with business.

Reply:
by confirmed with the business, they said to go with `DEFENCE` instead of `FULL_DEFENCE`?

Reply:
@timja-kainos That's correct. The claim store is still using `FULL_DEFENCE` though, as we made changes there before that, so that value still needs to be mapped in the model converter for now.